blackbox/docs/sql/conditional.txt

```diff
@@ -38,14 +38,13 @@ evaluate to NULL.
 
 ::
 
-    cr> select greatest(number_of_replicas, number_of_shards)
-    ...   from information_schema.tables
-    ...   where table_name='nodes';
-    +------------------------------------------------+
-    | greatest(number_of_replicas, number_of_shards) |
-    +------------------------------------------------+
-    | 1                                              |
-    +------------------------------------------------+
+    cr> select greatest(1, 2)
+    ...   from sys.nodes;
```

Comment:
if you're just using literals, using sys.cluster is maybe preferred, its always results to 1 row, while rowcount of sys.nodes depends on number of nodes